CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Support OpenCTI version 6.6.11 - No graphql changes
+- Support OpenCTI version 6.6.11 - No graphql changes
```

Comment:
does not look legit

Reply:
Yes, it's a bug, the job should not have been run since there was no new OpenCTI version, see https://github.com/weisshorn-cyd/gocti/pull/76 for a fix